enf-reducer/promote-common-constraints.metta

```diff
@@ -0,0 +1,42 @@
+(= (promoteCommonLiterals $expr)
+     (case (car-atom $expr)
+        (
+                (AND
```

Comment:
Best if you decrease the indentation level of this `(AND` to match it's closing bracket indentation level :)